pkg/lib/patternfly/patternfly-6-overrides.scss

```diff
@@ -469,3 +469,16 @@ to wrap around when there isn't enough space.
   // the correct sizing (becomes 10px on compact table, 18px on normal table)
   padding-block-start: calc(var(--pf-v6-c-table--cell--PaddingBlockStart) + 0.125rem)
 }
+
+// NOTE @Venefilyn: Patternfly has a weird issue with modals and skip to content
```

Comment:
Let's remove the "@Venefily" here, I'd say.  Just a link to the PF bug should be enough, with a "HACK" marker.

Reply:
Is HACK something we use in general? I didn't spot our guidance on it but there are a bunch of different anchors for these things like FIXME, NOTE, HACK (mostly used in python land), TODO, or SECTION. I'll change it though

Reply:
It's nothing super official, I think, and we don't really do anything with it (like grepping for it and checking whether bugs are closed). But we seem to gravitate towards "HACK" for bug workarounds. "FIXME" would be for our own bugs as opposed to workarounds for bugs in our dependencies.